config-schema.json

```diff
@@ -921,7 +932,6 @@
             },
             "updateUrl": {
               "type": "string",
-              "format": "url",
```

Comment:
The format specifier may not be removed. It makes sure, that the string conforms to an URL and that the user can't enter an invalid format.

An issue might be that `url` is not supported by all json tools, since that's an area of proposals and recommendations instead of well defined specifications...
The ajv validator supports it, others may not. As I just found out, that the cpp validator we are using just ignores it! Therefore I'll change the format to the following alternative:

```json
{
    "type": "string",
    "format": "uri",
    "pattern": "^(https?)://"
}
```

Reply:
Ok doesn't work anymore :-( ajv was happy last year with both uri and url formats!
This is exactly why I don't like JSON and JavaScript. No standard, suddenly after a tool or dependency update it's no longer working...

I'll replace it with a pure regex then

Reply:
Reason: ajv put the formats in an external library
https://ajv.js.org/docs/validation.html#formats
Now: how to use that in the command line version?

Reply:
Fixed in master branch: ajv-cli version downgraded to 3.3.0